client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx

```diff
@@ -4,7 +4,7 @@
 import { Card, Button } from '@automattic/components';
 import { isDesktop, isWithinBreakpoint, subscribeIsWithinBreakpoint } from '@automattic/viewport';
 import { translate } from 'i18n-calypso';
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useState, useReducer } from 'react';
```

Comment:
Please, confirm that `useState` is never used.

```suggestion
import React, { useEffect, useReducer } from 'react';
```

Reply:
Good spot. Fixed.